starboard/android/shared/android_main.cc

```diff
@@ -235,20 +237,19 @@ void* ThreadEntryPoint(void* context) {
 
   // Mark the app running before signaling app created so there's no race to
   // allow sending the first AndroidCommand after onCreate() returns.
-  g_app_running = true;
+  g_app_running.store(true);
 
   // Signal GameActivity_onCreate() that it may proceed.
   g_app_created_semaphore->Put();
 
   // Enter the Starboard run loop until stopped.
-  int error_level =
-      app.Run(std::move(command_line), GetStartDeepLink().c_str());
-#endif  // SB_API_VERSION >= 15
+  error_level = app.Run(std::move(command_line), GetStartDeepLink().c_str());
```

Comment:
why this change? any benefit to split it into 2 lines?

Reply:
Oh, leftover from another change that I removed. I'll undo this.